prow/plugins/approve/approvers/owners.go

```diff
@@ -556,6 +556,14 @@ func (ap Approvers) ListNoIssueApprovals() []Approval {
 	return approvals
 }
 
+func (ap Approvers) AssignedCCs() []string {
```

Comment:
Should we have unit test for these 2 new functions?

Reply:
nit: quick doc string for public functions